src/openzeppelin/access/ownable.cairo

```diff
@@ -5,48 +5,101 @@
 
 from starkware.cairo.common.cairo_builtins import HashBuiltin
 from starkware.starknet.common.syscalls import get_caller_address
+from starkware.cairo.common.math import assert_not_zero
+
+#
+# Events
+#
+
+@event
+func OwnershipTransferred(previousOwner: felt, newOwner: felt):
+end
+
+#
+# Storage
+#
 
 @storage_var
-func Ownable_owner() -> (owner: felt):
+func _owner() -> (owner: felt):
 end
 
+#
+# Constructor
+#
+
 func Ownable_initializer{
-        syscall_ptr : felt*, 
+        syscall_ptr : felt*,
         pedersen_ptr : HashBuiltin*,
         range_check_ptr
     }(owner: felt):
-    Ownable_owner.write(owner)
+    _transferOwnership(owner)
     return ()
 end
 
-func Ownable_only_owner{
-        syscall_ptr : felt*, 
+#
+# Protector (Modifier)
+#
+
+func Ownable_onlyOwner{
```

Comment:
this function is not meant to be exposed by any contract, therefore camelCase shouldn't be used here either